### ISSUE TITLE
Multiples appel de la route /prospect-zones

### ISSUE THREAD

**Issue by TheoLechemia** [PnX-SI/gn_module_flore_prioritaire#46]
Sur la carte-liste des ZP, l'appelle au webservice est fait 4 fois ! 
![Capture d’écran du 2023-05-25 11-05-49](https://github.com/PnX-SI/gn_module_flore_prioritaire/assets/15265745/24e95346-375e-4bb7-8f51-6d57afc01a72)

**Comment by jpm-cbna** [comment]
@TheoLechemia effectivement, j'ai le même problème sur l'instance déployée sur floresentinelle.fr...
